CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Unreleased
 
 [...]
+- **[FIX]** Remove hover color & pointer when `disabled` is passed to `ItemCheckbox`
```

Comment:
I guess this should also be applied to `ItemRadio`

Reply:
Yep, at least create a ticket if you prefer doing it separatly @mcellier

Reply:
Done